<launch>
  <!-- 跟随控制：默认直接订阅视觉状态 /person_track/vision_state -->
  <node pkg="kuavo_person_follow"
        type="follow_controller_node.py"
        name="follow_controller"
        output="screen">
    <param name="state_topic" value="/person_track/vision_state" />
    <param name="dist_target" value="1.5" />
    <param name="v_max" value="0.6" />
    <param name="yaw_rate_max" value="0.8" />
  </node>
</launch>
